nuget/runners/nunit.console-runner-with-extensions.nuspec

```diff
@@ -34,7 +34,7 @@
         <dependency id="NUnit.Extension.VSProjectLoader" version="$version$" />
         <dependency id="NUnit.Extension.NUnitV2ResultWriter" version="$version$" />
         <dependency id="NUnit.Extension.NUnitV2Driver" version="$version$" />
-        <dependency id="NUnit.Extension.TeamCityEventListener" version="1.0.1" />
+        <dependency id="NUnit.Extension.TeamCityEventListener" version="[1.0.*,1.1)" />
```

Comment:
This will cause users installing for the first time to get the lowest version that matches, unless they use options on the command line. See general comment.